locales/en/translation.json

```diff
@@ -21,7 +21,6 @@
   "Continue": "Continue",
   "Copied to clipboard": "Copied to clipboard",
   "Copy Link": "Copy Link",
-  "Copyright": "Copyright",
```

Comment:
@davisagli Oh, I guess this could also be removed from other translations if we wanna keep them clean/synced.

Reply:
Sure. But it's not super important, because it will presumably be removed as part of the next round of work by the translators if it's not in the English dev translations.